### PR TITLE
fix: preserve case for stream descriptors

### PR DESCRIPTION
**Pull request by cosmin**
The accidental tolower in `SplitAndTrimSkipEmpty` was causing stream descriptors to not preserve case for certain things like accessibilities.

### CODE REVIEW

**Comment by xavierlaffargue** [comment]
@joeyparrish can you add @cosmin as maintainer for Shaka packager? 
a single maintainer is not enough and causes a bottleneck.

**Comment by cosmin** [comment]
I am a maintainer but every change should get reviewed anyway so I'm waiting for @joeyparrish to get back and approve a few pull requests that are blocking a 3.0 release.

**Review by joeyparrish — APPROVED** [review]

**Pull request merged**
